### PR TITLE
test: add install lxcfs script for CentOS

### LINKED ISSUE #567: [bug] test fail to install lxcfs in CentOS
**Issue Description**

In hack/make.sh, using `apt-get` to install lxcfs.
It fails on CentOS, need to fix.

**Describe what happened**:

**Describe what you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:
1.
2.
3.

**Anything else we need to know?**:

**Environment**:
- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by Letty5411**
Signed-off-by: letty <letty.ll@alibaba-inc.com>

**1.Describe what this PR did**
Install lxcfs on CentOS from source code.
**2.Does this pull request fix one issue?** 

fix #567 
**3.Describe how you did it**

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

**Comment by allencloud** [comment]
Please help review this, @zzchun 
Thanks a lot.

File: hack/make.sh [hunk 1]
```diff
@@ -30,9 +30,13 @@ function install_pouch ()
 	
 	# install lxcfs
 	echo "Install lxcfs"
-	apt-get install lxcfs
-	# MUST stop lxcfs service, so pouchd could take over it.
-	service lxcfs stop
+	if grep -qi "ubuntu" /etc/issue ; then
+		apt-get install lxcfs
```
Comment (zzchun) [inline hack/make.sh]: In ubuntu:14.04 `apt-get install lxcfs` will fail, as lollows
```
root@xxx:~# apt-get isntall lxcfs 
E: Invalid operation isntall
root@host14:~# apt-get install lxcfs 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package lxcfs
```

os:
Linux host14 4.4.0-93-generic #116~14.04.1-Ubuntu SMP Mon Aug 14 16:07:05 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

File: hack/install_lxcfs_on_centos.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+yes | yum install autotools-dev m4 autoconf2.13 autobook autoconf-archive gnu-standards autoconf-doc libtool
+yes | yum install fuse-devel.$(uname -p)
+yes | yum install pam-devel.$(uname -p)
+
+TMP=$(mktemp -d)
+trap "rm -rf $TMP" EXIT
+
+cd $TMP &&
+git clone https://github.com/lxc/lxcfs.git &&
```
Comment (zzchun) [inline hack/install_lxcfs_on_centos.sh]: Should checkout to a stable branch  or commit_id

Comment (allencloud) [reply hack/install_lxcfs_on_centos.sh]: I agree with what @zzchun said.

**Comment by zzchun** [comment]
LGTM

**Comment by allencloud** [comment]
Thanks for your work @Letty5411 and review @zzchun 
Merging...

**Pull request merged**